django_cloud_deploy/skeleton/templates/settings_template/extra_settings.py-tpl

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .base_settings import *
+
+
+if 'INSTALLED_APPS' in locals():
+    INSTALLED_APPS += [
+        'cloud_admin.apps.CloudAdminConfig',
+    ]
+
+if 'TEMPLATES' not in locals():
+    TEMPLATES = []
+
+TEMPLATES += [
+    {
```

Comment:
What does this do?

Reply:
W created a directory to save the templates used to overwrite the admin app. This will make those templates available.

Reply:
Could you add that as a comment? Maybe:
```suggestion
# Add the templates required by CloudAdminSite.
```